include/arch/arm/arch/64/mode/model/statedata.h

```diff
@@ -39,7 +39,7 @@ extern hw_asid_t armKSNextASID VISIBLE;
 #endif
 
 #ifdef CONFIG_KERNEL_LOG_BUFFER
-pde_t *armKSGlobalLogPDE;
+pde_t *armKSGlobalLogPTE;
```

Comment:
Isn't this missing a `extern` keyword?

Reply:
Yes, and the other one is declared as:

https://github.com/seL4/seL4/blob/46f2b4d255bbf50e593103a859d3cf702a5232b6/src/arch/arm/64/model/statedata.c#L93

There's a `typedef pte_t pde_t;` in include/arch/arm/arch/64/mode/object/structures.h, but still.